Source/WebCore/html/ColorInputType.cpp

```diff
@@ -34,6 +34,7 @@
 
 #include "AXObjectCache.h"
 #include "CSSPropertyParserConsumer+Color.h"
+#include "CSSPropertyParserConsumer+ColorInlines.h"
```

Comment:
How do you feel about omitting the redundant include in a case like this?
```suggestion
#include "CSSPropertyParserConsumer+ColorInlines.h"
```

Reply:
Seems reasonable to me.